src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -384,4 +430,323 @@ private static String trimExcessWhitespaces(String text) {
         return result.toString();
     }
 
+    /**
+     * Finds and return if summary tag present.
+     *
+     * @param javadoc Javadoc root node.
+     * @return true, if first sentence contains @summary tag.
+     */
+    private static boolean containsSummaryTag(DetailNode javadoc) {
+        boolean contains = false;
+        for (DetailNode node : javadoc.getChildren()) {
+            if (node.getType() == JavadocTokenTypes.JAVADOC_INLINE_TAG) {
+                final DetailNode[] child = node.getChildren();
+                if (child[1].getType() == JavadocTokenTypes.CUSTOM_NAME
```

Comment:
are we sure that there will be no exception ?
should we check for size of child to be more then 1 ?

Reply:
Sir, this is not required as there will always be atleast two token for a inline Tag, which are `RCURLY` and `LCURLY` , so it is assured that it will not give any NLP.

Reply:
add comment

Reply:
Done.